wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -887,6 +887,11 @@
 							<xs:documentation>OAuth2 client credentials grant flow per RFC 6749</xs:documentation>
 						</xs:annotation>
 					</xs:enumeration>
+					<xs:enumeration value="OIDC2AuthorizationCode">
+						<xs:annotation>
+							<xs:documentation>OpenID Connect authorization code flow per Open ID Connect Core</xs:documentation>
+						</xs:annotation>
```

Comment:
change as-> OpenID Connect authorization code flow as per Open ID Connect Core.
Same needs update for OAuthAuthorizationCode, OAuthClientCredentials enum types description and also needs update in the spec document Table 6: Authorization server configuration types.